rust-runtime/inlineable/src/event_stream.rs

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+use smithy_eventstream::error::Error;
+use smithy_eventstream::frame::{Header, Message};
+use smithy_eventstream::str_bytes::StrBytes;
+
+pub struct ResponseHeaders<'a> {
+    pub content_type: &'a StrBytes,
+    pub message_type: &'a StrBytes,
+    pub smithy_type: &'a StrBytes,
+}
+
+fn expect_header_str_value<'a>(
```

Comment:
any reason to not add this to smithy-eventstream?

Reply:
No good reason. Moved.